package.json

```diff
@@ -14,11 +14,11 @@
     "@edx/frontend-enterprise-logistration": "8.0.0",
     "@edx/frontend-enterprise-utils": "8.0.0",
     "@edx/frontend-platform": "7.1.0",
-    "@edx/openedx-atlas": "^0.6.0",
-    "@edx/reactifex": "^2.2.0",
+    "@edx/openedx-atlas": "0.6.0",
```

Comment:
[inform] Ensure `npm install` is run to update the package-lock.json file as well 😄